notebooks/week3/02_model_serving

```diff
@@ -0,0 +1,169 @@
+# Databricks notebook source
+# MAGIC %pip install ../housing_price-0.0.1-py3-none-any.whl
```

Comment:
this is an example for installing custom package in a notebook session - which can be also taken from Volumes.  you need to change this with your own custom package - taxinyc

Reply:
Hello Basak,
i have the .whl file installed in my cluster. Thats why i just commented it out. Somehow i have problems installing in via pip when not in the same path. 
That is why i did it that way.
Best wishes 
Maren